sdk/consumption/Microsoft.Azure.Management.Consumption/tests/ScenarioTests/BudgetTests.cs

```diff
@@ -23,7 +23,7 @@ public class BudgetTests : TestBase
         [Fact]
         public void BudgetCreateOrUpdateTest()
         {
-            using (MockContext context = MockContext.Start(this.GetType().FullName))
+            using (MockContext context = MockContext.Start(this.GetType().Name))
```

Comment:
I assume this will cause issues if there are multiple classes with the same test name correct?

Reply:
Yes, you are right. I did not encounter that in any of the ones I worked on.